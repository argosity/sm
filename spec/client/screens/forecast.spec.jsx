import { React, Snapshot, getScreenInstance } from 'hippo/testing/index'; // eslint-disable-line no-unused-vars
import Forecast from 'sm/screens/forecast';

const screenInstance = getScreenInstance('forecast');

describe('Screen Forecast', () => {
    it('renders and matches snapshot', () => {
        const fc = mount(<Forecast screen={screenInstance} />);
        expect(fc).toHaveRendered('Forecast');
        expect(Snapshot(<Forecast screen={screenInstance} />)).toMatchSnapshot();
    });
});
