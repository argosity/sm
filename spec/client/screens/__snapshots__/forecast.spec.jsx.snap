// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen Forecast renders and matches snapshot 1`] = `
.c2 {
  display: inline-block;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  fill: #666666;
  stroke: #666666;
}

.c2 g {
  fill: inherit;
  stroke: inherit;
}

.c2 *:not([stroke])[fill="none"] {
  stroke-width: 0;
}

.c2 *[stroke*="#"],
.c2 *[STROKE*="#"] {
  stroke: inherit;
  fill: none;
}

.c2 *[fill-rule],
.c2 *[FILL-RULE],
.c2 *[fill*="#"],
.c2 *[FILL*="#"] {
  fill: inherit;
  stroke: none;
}

.c0 {
  box-sizing: border-box;
  cursor: pointer;
  outline: none;
  font: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  font: inherit;
  margin: 0;
  background-color: transparent;
  overflow: visible;
  text-transform: none;
  text-align: center;
  display: inline-block;
  min-width: 96px;
  max-width: 384px;
  font-weight: 600;
  border: 2px solid transparent;
  border-radius: 24px;
  color: #444444;
  font-size: 1.1875rem;
  line-height: 24px;
  padding: 10px 46px;
}

.c0:hover {
  box-shadow: 0px 0px 0px 2px transparent;
}

.c0:hover svg {
  fill: #000000;
  stroke: #000000;
  -webkit-transition: none;
  transition: none;
}

.c3:first-child:not(:last-child) {
  margin-right: 12px;
}

.c1 {
  display: inline-block;
}

.c1:first-child:not(:last-child) {
  margin-right: 12px;
}

.c1 > * {
  vertical-align: bottom;
}

@media only screen and (min-width:700px) {
  .c0 {
    -webkit-transition: 0.1s ease-in-out;
    transition: 0.1s ease-in-out;
  }
}

<div
  className="hSKpsq"
>
  <div
    className="hippo-screen forecast is-active StyledGrid-khEPOT cOEAnA"
    data-screen-id="forecast"
    rows={
      Array [
        "xxsmall",
        "flex",
        "xsmall",
      ]
    }
  >
    <div
      aria-label={undefined}
      className="hippo-toolbar sc-dnqmqq dDLAIh StyledBox-bStriS bFIgXI"
      direction="row"
    >
      <div
        className="StyledBox__StyledBoxGap-ixLlSs laXbkr"
        direction="row"
      />
      <button
        aria-label={undefined}
        className="grommetux-control-icon-search c0"
        color="transparent"
        disabled={false}
        href={undefined}
        icon={<Search />}
        label="Click to find Show"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        type="button"
      >
        <span
          aria-hidden={true}
          className="c1"
        >
          <svg
            aria-label="Search"
            className="c2"
            height="24px"
            role="img"
            version="1.1"
            viewBox="0 0 24 24"
            width="24px"
          >
            <path
              d="M15,15 L22,22 L15,15 Z M9.5,17 C13.6421356,17 17,13.6421356 17,9.5 C17,5.35786438 13.6421356,2 9.5,2 C5.35786438,2 2,5.35786438 2,9.5 C2,13.6421356 5.35786438,17 9.5,17 Z"
              fill="none"
              stroke="#000"
              strokeWidth="2"
            />
          </svg>
        </span>
        <span
          className="c3"
        >
          Click to find Show
        </span>
      </button>
    </div>
  </div>
</div>
`;
